style.css

```diff
@@ -63,6 +95,229 @@ h1 {
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+#about-me-text h2 {
+  font-size: 40px;
+  font-weight: bold;
+  color: #172b4d;
+}
+
+#about-me-text p {
+  font-size: 20px;
+  font-weight: 500;
+  color: #42526e;
+  padding: 0 20px 0 20px;
+  text-align: center;
+}
+
+#about-me-text button {
+  background-color: #36b37f;
+  color: white;
+  font-size: 20px;
+  outline-width: 0;
+  font-weight: 400;
+  align-self: center;
+  padding: 15px;
+  border-radius: 10px;
+  border: 0;
+}
+
+#about-me-skills {
+  width: 100%;
+  display: grid;
+  justify-content: center;
+  padding: 10px;
+  gap: 15px;
+  grid-template-columns: repeat(auto-fill, 327px);
+}
+
+#about-me-skills h4 {
+  color: #344563;
+  font-size: 24px;
+  margin-top: -10px;
+}
+
+#contact-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 20px;
+}
+
+#contact-section p {
+  font-size: 30px;
+  font-weight: bold;
+  padding: 0 25px 0 25px;
+  width: 100%;
+  text-align: center;
+}
+
+#form {
+  font-size: 30px;
+  font-weight: bold;
+  padding: 0 15px 0 15px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+#form input {
+  width: 100%;
+  height: 48px;
+  border: 1px solid #36b37f;
+  border-radius: 5px;
+}
+
+#form textarea {
+  width: 100%;
+  border: 1px solid #36b37f;
+  border-radius: 5px;
+}
+
+#form button {
+  padding: 10px 10px 10px 10px;
+  border-radius: 15px;
+  width: 129px;
+  height: 48px;
+  font-size: 17px;
+  border: none;
+  background-color: #36b37f;
+  color: white;
+  align-self: center;
+}
+
+.card {
+  position: relative;
+  width: 327px;
+  height: 474px;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: end;
+  border-radius: 10px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+}
+
+.skill-card {
+  width: 327px;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  border-radius: 10px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  gap: 15px;
+  padding: 25px 0 25px 0;
+}
+
+.image-placeholder {
+  width: 100%;
+  top: 0;
+  position: absolute;
+  left: 0;
+  height: 100%;
+}
+
+.card-content {
+  width: 100%;
+  height: 230px;
+  border-top-left-radius: 15px;
+  border-top-right-radius: 15px;
+  background-color: white;
+  z-index: 1;
+  padding: 2px;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+}
+
+.project-title {
+  font-size: 24px;
+  color: #3a4a42;
+  padding: 0 10px 0 10px;
+  margin-top: 5px;
+}
+
+.tag-language-container {
+  display: inline-flex;
+  justify-content: start;
+  flex-wrap: wrap;
+  gap: 12px;
+}
+
+.skill-language-container {
+  display: inline-flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 12px;
+  padding: 10px;
+}
+
+.tag-language {
+  padding: 7px;
+  background-color: #ebf0ee;
+  color: #3a4a42;
+  font-size: 16px;
+  font-weight: bold;
+  border-radius: 5px;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached images in the table.

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-04 14-58-31](https://user-images.githubusercontent.com/66476329/236189480-fc6b0bed-b0e4-477c-a89e-a6e3f05f9378.png)

</td>
<td>

![Screenshot from 2023-05-04 14-58-44](https://user-images.githubusercontent.com/66476329/236189510-0a1444c0-74f5-470f-b195-f5bad48601cb.png)

</td>
</tr>
</table>